kvbc/src/categorization/kv_blockchain.cpp

```diff
@@ -146,6 +149,15 @@ void KeyValueBlockchain::loadCategories() {
   }
 }
 
+void KeyValueBlockchain::addGenesisBlockKey(Updates& updates) const {
+  const auto stale_on_update = true;
+  updates.addCategoryIfNotExisting<VersionedInput>(kConcordInternalCategoryId);
```

Comment:
Just a que, should we add kConcordReconfigurationCategoryId and add the genesis block only for 
 for those blocks with kConcordReconfigurationCategoryId ?

Reply:
Actually, I am not sure what the intention of `kConcordReconfigurationCategoryId` is. Maybe it can be used instead. Maybe @yontyon can give us a hint.

Reply:
There are two `kvbc_key_types` files, it might be a bit confusing.

Reply:
@yontyon We call `updates.addCategoryIfNotExisting()`to add it to the updates only, not globally. Is that what you meant?

In terms of the ID, we use 0x32. Is that OK for `kConcordInternalCategoryId`?